src/components/dashboard/components/dictionary/EditDictionary.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import DictionaryModal from './common/DictionaryModal';
+import {
+  fetchingOrganizations,
+  editDictionary,
+} from '../../../../redux/actions/dictionaries/dictionaryActionCreators';
+import languages from './common/Languages';
+
+export class EditDictionary extends React.Component {
+  preSelectDefaultLocale = () => {
+    const { dictionary } = this.props;
+    for (let i = 0; i < languages.length; i++) { // eslint-disable-line
+      if (languages[i].value === dictionary.default_locale) {
+        return (<option value={languages[i].value} selected>
+          {languages[i].label}
+        </option>);
+      }
+    }
+    return null;
+  }
+
+  submit = (data) => {
+    const {
+      dictionary: {
+        url,
+      },
+    } = this.props;
+    return this.props.editDictionary(url, data);
+  }
+  render() {
+    return (
+      <DictionaryModal
+        title="Edit Dictionary"
+        buttonname="Update Dictionary"
+        show={this.props.show}
```

Comment:
Looking at this variable naming `show` it does not really communicate the essence of the value the variable is holding. Please improve the variable naming `show` to become something like `showModal` or any other name that properly communicates the purpose of the variable.

Reply:
I think it is sufficient for a modal, since hide and show are common characteristics of a modal. I thought the same too but since I am reusing an existing component. I can't rename the variable here and across the application as it does not pertain to the ticket I am working on. If there is any need for that, another ticket would be created.